profile/build.go

```diff
@@ -17,13 +17,53 @@
 package profile
 
 import (
+	"os"
+	"path/filepath"
+
 	"github.com/buildpacks/libcnb"
 )
 
 func Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 	// NOTE: the logger is not passed into this function, that will likely be a change in libcnbv2
-
 	result := libcnb.NewBuildResult()
-	// TODO: implement build actions
+
+	layer, err := context.Layers.Layer(profileName)
+
+	if err != nil {
+		return result, err
+	}
+
+	execPath := layer.Exec.FilePath(execDScriptName)
+
+	err = os.MkdirAll(layer.Exec.Path, os.ModePerm)
+	if err != nil {
+		return result, err
+	}
+
+	execDScript, err := os.ReadFile(filepath.Join(context.Buildpack.Path, "execd_wrapper.sh"))
```

Comment:
```suggestion
	execDScript, err := os.ReadFile(filepath.Join(context.Buildpack.Path, "scripts", "execd_wrapper.sh"))
```

Read from `scripts/` directory where it get's packaged (if you make the other changes I've suggested).